README-Mod.md

```diff
@@ -0,0 +1,194 @@
+<!--
+ Copyright 2023 Interlynk.io
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# `sbomqs`: Quality metrics for your sbom's 
+---
+[![Go Reference](https://pkg.go.dev/badge/github.com/interlynk-io/sbomqs.svg)](https://pkg.go.dev/github.com/interlynk-io/sbomqs)
+[![Go Report Card](https://goreportcard.com/badge/github.com/interlynk-io/sbomqs)](https://goreportcard.com/report/github.com/interlynk-io/sbomqs)
+
+`sbomqs` is your primary tool to assess the quality of your sboms. The higher the score the more consumable your sboms are. 
+
+```sh
+go install github.com/interlynk-io/sbomqs@latest
+```
+other installation [options](#installation).
+
+# What is a good quality SBOM
+A good quailty SBOM should allow for managements of assets, license, vulnerabilities, Intellectual Property, configuration management and incidence response. 
+
+A quality SBOM is one that is accurate, complete, and up-to-date.There are many factors that go into constructing a good high quality sbom
+1. Identify & list all components of your product along with their transitive dependencies. 
+2. List all your components along with their versions & content checksums. 
+3. Include accurate component licenses. 
+4. Include accurate lookup identifiers i.e Purls/CPE. 
+5. Quality SBOM depends a lot upon which stage of the lifecycle it was generated at, we believe closer to the build time is ideal. 
```

Comment:
it ~was~ has been generated at, and, we belive closer to the build is the ideal time.